docs/reference/filters.md

```diff
@@ -871,3 +871,108 @@ Example:
 ```
 auditLog()
 ```
+
+## apiUsageMonitoring
+
+The `apiUsageMonitoring` filter adds API related metrics to the Skipper monitoring. It is by default not activated. Activate
+it by providing the `-enable-api-usage-monitoring` flag at Skipper startup. In its deactivated state, it is still
+registered as a valid filter (allowing route configurations to specify it), but will perform no operation. That allows,
+per instance, production environments to use it and testing environments not to while keeping the same route configuration
+for all environments.
+
+NOTE: Make sure to activate the metrics flavour proper to your environment using the `metrics-flavour`
+flag in order to get those metrics.
+
+Example:
+```bash
+skipper -enable-api-usage-monitoring -metrics-flavour prometheus
+```
+
+The structure of the metrics is:
+
+    apiUsageMonitoring.custom.<Application ID>.<API ID>.<HTTP Verb>.<Path Template>.<Metric Name>
+
+The available metrics are:
+
+* HTTP exchanges counting:
+    * **http_count**: the number of HTTP exchanges
+    * **http5xx_count**: number of HTTP exchanges resulting in a server error (HTTP status in the 500s)
+    * **http4xx_count**: number of HTTP exchanges resulting in a client error (HTTP status in the 400s)
+    * **http3xx_count**: number of HTTP exchanges resulting in a redirect (HTTP status in the 300s)
+    * **http2xx_count**: number of HTTP exchanges resulting in success (HTTP status in the 200s)
+* Timing:
+    * **latency**: time between the first observable moment (a call to the filter's `Request`) until the last (a call to the filter's `Response`) 
+
+Endpoints can be monitored using the `apiUsageMonitoring` filter in the route. It accepts JSON objects (as strings)
```

Comment:
The docs could be less puzzling if the below config example was JSON and not YAML ;)

Reply:
my bad. I realize that the yaml is the spec of the JSON.